pkg/transferer/load.go

```diff
@@ -20,6 +20,8 @@ const (
 	LoadStatusOK
 	LoadStatusWait
 	LoadStatusTerminate
+
+	MaxWaitStatusAttempts int = 10
```

Comment:
it's not attempts limit, please rename this to something more meaningful

`MaxWaitStatusInSequence` maybe?

Reply:
Applies to `waitAttemptsCounter` as well

Reply:
the latter could be just `waitStatusCounter`

Reply:
Resolved in d2bdbaca363c55fe9de7934d01ac8667233954f2